plugins/test_control_plugin/test_control_plugin.cpp

```diff
@@ -79,6 +92,8 @@ void test_control_plugin_impl::process_next_block_state(const chain::block_state
       _producer_sequence = -1;
```

Comment:
Seems like the code would be clearer if `_producer_sequence` was initialized to 0 and not have all the `_producer_sequence + 1` throughout this code. Would allow you to remove the comment you added `_producer_sequence is index, aligning it with slot number`.

Reply:
it should be aligned with `_where_in_sequence` and later is used in external interface so I'll need to change it in few tests.

Reply:
Sounds good.